### ISSUE TITLE
node-chrome-debug container exits if debug is set to true

### ISSUE THREAD

**Issue by testphreak** [SeleniumHQ/docker-selenium#463]
## Meta -
Image(s):  
node-chrome-debug

Image Version:  
3.3.1-cesium

Docker version:
17.04.0-ce, build 4845c56

OS: 
Ubuntu 16.04.2 LTS

## Expected Behavior -
Grid is up and running and debug logs are available

## Actual Behavior -
node-chrome-debug container exits after a few seconds. Don't see this issue with node-chrome.

docker-compose file -

```
selhub:
  image: selenium/hub:3.3.1-cesium
  environment:
    - GRID_MAX_SESSION=5
    - SE_OPTS=-debug true
  ports:
    - 4444:4444

nodechromedebug:
  image: selenium/node-chrome-debug:3.3.1-cesium
  privileged: true
  environment:
    - NODE_MAX_INSTANCES=5
    - NODE_MAX_SESSION=5
  volumes:
    - /dev/shm:/dev/shm
  ports:
    - 5900
  links:
    - selhub:hub
```

When doing `docker ps -a` I see -

```
CONTAINER ID        IMAGE                                     COMMAND                  CREATED             STATUS                      PORTS                    NAMES
450284916d1c        selenium/node-chrome-debug:3.3.1-cesium   "/opt/bin/entry_po..."   20 minutes ago      Exited (0) 20 minutes ago                            scripts_nodechromedebug_1
7b09c520864a        selenium/hub:3.3.1-cesium                 "/opt/bin/entry_po..."   20 minutes ago      Up 20 minutes               0.0.0.0:4444->4444/tcp   scripts_selhub_1
```

**Comment by testphreak** [comment]
Not sure if this issue has anything to do with issue #423

**Comment by testphreak** [comment]
This issue appears resolved with 3.4.0-actinium.
